src/build/SharedDeps.zig

```diff
@@ -261,6 +261,10 @@ pub fn add(
         try static_libs.append(breakpad_dep.artifact("breakpad").getEmittedBin());
     }
 
+    // Layer shell is only included if we're doing a GTK Wayland build,
+    // but we should offer the option here
+    _ = b.systemIntegrationOption("gtk4-layer-shell", .{});
```

Comment:
This should go in `src/build/Config.zig` and not here (see there for a comment on why)